spaces/domains.json

```
@@ -7,6 +7,7 @@
   "governance.cyberestate.io": "cyberrealestate.eth",
   "gov.meritvault.xyz": "meritvault.eth",
   "vote.hifi.finance": "hifi-finance.eth",
+  "dao.general.rebellionprotocol.com": "rebellionprotocol.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "dao.general.rebellionprotocol.com": "rebellionprotocol.eth",
```